lib/clone.js

```diff
@@ -0,0 +1,22 @@
+const fs = require('fs')
```

Comment:
Why split code into multiple files? Not saying wrong but for such a small script it seems overkill.